test/Client.test.ts

```diff
@@ -124,6 +124,68 @@ describe('canMessage', () => {
   })
 })
 
+describe('canMessageBatch', () => {
+  it('can confirm multiple users are on the network statically', async () => {
+    // Create 10 registered clients
+    const registeredClients = await Promise.all(
+      Array.from({ length: 10 }, () => newLocalHostClient())
+    )
+    // Wait for all clients to be registered
+    await Promise.all(
+      registeredClients.map((client) => waitForUserContact(client, client))
+    )
+    // Now call canMessage with all of the peerAddresses
+    const canMessageRegisteredClients = await Client.canMessage(
+      [...registeredClients.map((client) => client.address)],
```

Comment:
Think this line could be simplified to
`registeredClients.map((client) => client.address)`